src/components/WidgetStats/WidgetStatsPresentation.js

```diff
@@ -13,7 +13,7 @@ function WidgetStatsPresentation({ data }) {
     : 'text-4xl md:text-5xl lg:text-6xl font-medium opacity-0'
   const labelStyling = 'text-base font-thin'
   return (
-    <section className="bg-tertiaryA pt-3 md:pt-6">
+    <section className="bg-tertiaryA pt-3 md:pt-6 hidden">
```

Comment:
This solution using css would effectively hide the widget, but it would still load the component and as such execute any data fetching that the widget requires. Consider instead preventing these widget components from loading entirely here: `src/components/Widget/WidgetPresentation.js`